### PR TITLE
UtBS Balance

### PR DESCRIPTION
**Pull request by Hejnewar**
As in title, some bug fixes, balance and some more balance.

S1 - Way too easy. I still comfortably won S1 after changes. 
S4 - Just slightly too hard in my taste no major. 
S5 - One of the two scenarios that actually got harder with the update. Maybe I even made it bit too easy but will see later. 
S11 - The second harder scenario. Also perhaps too easy now, maybe Im just playing it somewhat unoptimally, will see if I can find better way. 

Dust Devil - The amlas were joke for this kind of xp. 
L1 units - just price updates in later scenarios. 
Stalwart and Protector - I found it way more fun when there was a unit that is very strong, very expensive and takes 5-6 scenarios to level up. Also not at all op. 
Sun Ladies - general buffs, when I generally though of them I didnt take into consideration some things and now they came out so buffs. 

Bunch of AMLA updates for heroes. 
Bunch of ability fixes. 

Return to 100/150/200% xp. 306xp for lvl 3 is the way. 

Ideally merged before next version, no string changes.

### CODE REVIEW

**Comment by newfrenchy83** [comment]
@Hejnewar experience in advance formula must be adjusted for level change

**Comment by Hejnewar** [comment]
Ah yes forgot about that, Will fix in a minute.

**Comment by newfrenchy83** [comment]
> Ah yes forgot about that, Will fix in a minute.

`    [effect]
        [filter]
            level=1
#ifdef EASY
            formula="max_experience > 24"
#endif
#ifdef NORMAL
            formula="max_experience > 32"
#endif
#ifdef HARD
            formula="max_experience > 42"
#endif
        [/filter]
        apply_to=level
        set=2
    [/effect]
    [effect]
        [filter]
            level=2
#ifdef EASY
            formula="max_experience > 41"
#endif
#ifdef NORMAL
            formula="max_experience > 50"
#endif
#ifdef HARD
            formula="max_experience > 64"
#endif
        [/filter]
        apply_to=level
        set=3
    [/effect]`

and the code

**Comment by newfrenchy83** [comment]
> Ah yes forgot about that, Will fix in a minute.

for level 1 to 2 NORMAL experience=32 HARD experience=42

for level 2 to 3 NORMAL experience=50 HARD experience=64

File: data/campaigns/Under_the_Burning_Suns/units/quenoth/Youth.cfg [hunk 1]
```diff
@@ -679,16 +700,12 @@ _"Nym is Kaleh’s childhood friend, a young, rebellious lady with quick wits an
...
             icon = attacks/bolas.png
             [specials]
                 {WEAPON_SPECIAL_USES_ATTACKS 1}
             [/specials]
-            attacks_used = 1
-        [/effect]
-        [effect]
-            apply_to = max_attacks
-            increase = 1
+            attacks_used = 2
```
Comment (stevecotton) [inline data/campaigns/Under_the_Burning_Suns/units/quenoth/Youth.cfg:708]: This still has `{WEAPON_SPECIAL_USES_ATTACKS 1}`

Comment (Hejnewar) [reply data/campaigns/Under_the_Burning_Suns/units/quenoth/Youth.cfg:708]: Meh... but yeah Im prone to making mistakes like that. At least it wasnt anything big. (This time.)

**Pull request merged**